README.md

```diff
@@ -82,6 +82,16 @@ npm publish --access=public
 
 NPM will automatically build the library first, as `package.json` defines `script/prepare`.
 
+#### Production Release
+To deploy a production release:
+
+1. Merge all changes to `master`
+2. Use https://github.com/concord-consortium/portal-report/releases to create a new release tag
```

Comment:
At least in the past this approach hasn't been safe. When you make a new release tag directly in GitHub it makes a lightweight tag. And our s3-deploy code can get confused (at least when it was running on travis) by lightweight tags.

So to be safe you should make a tag in your local git client (commandline or GUI) that has a description associated with it. I usually put at least the version number in this description. Then push this tag to production. Now after making this "heavy weight" tag you can go to Github and make a release and choose this tag.

Reply:
This is straight from the starter project so if this is true, then most of our documents release procedures are wrong:
https://github.com/concord-consortium/starter-projects#deployment

Reply:
So is the "annotated" tag what you are referring to?
https://git-scm.com/book/en/v2/Git-Basics-Tagging

Reply:
Is this what you're thinking?
```
3. Make a new version tag using your local git client with the version number in the description (e.g., `git tag -a v1.1.1 -m "version 1.1.1"`) and push the tag to the portal-report repo (e.g., `git push origin v1.1.1`).
4. Use https://github.com/concord-consortium/portal-report/releases to create a new release using the newly created tag.
```

Reply:
Yes an annotated tag is correct. 

The starter projects description uses a production branch, so it doesn't matter much if the tag gets deployed or not. But for something like the portal-report or question interactives it does mater that the tag is deployed correctly. I thought I had corrected this in the starter projects before. I do try to point it out whenever people talk about using Github to create tags.

Reply:
Makes sense.  I'll push the above changes.

Reply:
Steps updated.  See the latest commit.

Reply:
Codecov doesn't like the lack of readme tests! 🤣

Reply:
:) It is actually that there is a random difference when cypress runs the tests. Sometimes it causes a scroll event and sometimes it doesn't: https://app.codecov.io/gh/concord-consortium/portal-report/compare/335/changes

So when the scroll event doesn't happen not as much code is covered.